test/organizations/CompletionTracking.js

```diff
@@ -0,0 +1,197 @@
+			expect(noTrackingDisplay.isProgressDisplayed).to.be.false;
+		});
+		it('actions missing', () => {
+			expect(noActions.isCompletionTracked).to.be.undefined;
+			expect(noActions.isProgressDisplayed).to.be.undefined;
+		});
+	});
+
+	describe('organization get completion action', () => {
+		it('tracking, no display', () => {
+			expect(trackingNoDisplay._getCompletionAction(true)).to.be.undefined;
+			expect(trackingNoDisplay._getCompletionAction(false).name).to.equal('do-not-track-completion');
+		});
+		it('tracking with display', () => {
+			expect(trackingAndDisplay._getCompletionAction(true)).to.be.undefined;
+			expect(trackingAndDisplay._getCompletionAction(false).name).to.equal('do-not-track-completion');
+		});
+		it('no tracking, no display', () => {
+			expect(noTrackingDisplay._getCompletionAction(true).name).to.equal('track-completion');
+			expect(noTrackingDisplay._getCompletionAction(false)).to.be.undefined;
+		});
+		it('no actions', () => {
+			expect(noActions._getCompletionAction(true)).to.be.undefined;
+			expect(noActions._getCompletionAction(false)).to.be.undefined;
+		});
+	});
+
+	describe('organization get display action', () => {
+		it('tracking, no display', () => {
+			expect(trackingNoDisplay._getDisplayAction(true).name).to.equal('display-progress');
+			expect(trackingNoDisplay._getDisplayAction(false)).to.be.undefined;
+		});
+		it('tracking with display', () => {
+			expect(trackingAndDisplay._getDisplayAction(true)).to.be.undefined;
+			expect(trackingAndDisplay._getDisplayAction(false).name).to.equal('do-not-display-progress');
+		});
+		it('no tracking, no display', () => {
+			expect(noTrackingDisplay._getDisplayAction(true).name).to.equal('display-progress');
+			expect(noTrackingDisplay._getDisplayAction(false)).to.be.undefined;
+		});
+		it('no actions', () => {
+			expect(noActions._getDisplayAction(true)).to.be.undefined;
+			expect(noActions._getDisplayAction(false)).to.be.undefined;
+		});
+	});
+
+	describe('update actions', () => {
+		let sandbox, trackingSpy, displaySpy, display;
+
+		beforeEach(() => {
+			sandbox = sinon.sandbox.create();
+			display = trackingNoDisplay;
+			trackingSpy = sandbox.spy(display, 'updateCompletionTracking');
+			displaySpy = sandbox.spy(display, 'updateDisplayProgress');
+		});
+
+		afterEach(() => {
+			sandbox.restore();
+		});
+
+		describe('tracking completion, not tracking display', () => {
+			it('correct parameters', () => {
+				display.updateCompletionTracking(false);
+				display.updateDisplayProgress(true);
+				expect(trackingSpy.returnValues[0]).to.be.a('promise');
+				expect(displaySpy.returnValues[0]).to.be.a('promise');
+			});
+			it('incorrect parameters', () => {
+				display.updateCompletionTracking(true);
+				display.updateDisplayProgress(false);
+				expect(trackingSpy.returnValues[0]).to.be.undefined;
+				expect(displaySpy.returnValues[0]).to.be.undefined;
+			});
+		});
+	});
+
+	describe('update actions', () => {
```

Comment:
there is already test above with title `'update actions'` . Could you come up with unique names?